full/src/main/java/apoc/ttl/TTLLifeCycle.java

```diff
@@ -38,8 +38,11 @@ public TTLLifeCycle(JobScheduler scheduler, GraphDatabaseAPI db, ApocConfig apoc
 
     @Override
     public void start() {
+        String apocTTLEnabledDb = String.format(ApocConfig.APOC_TTL_ENABLED_DB, this.db.databaseName());
+        boolean containsTTLEnabled = apocConfig.containsKey(apocTTLEnabledDb);
         boolean enabled = apocConfig.getBoolean(ApocConfig.APOC_TTL_ENABLED);
-        if (enabled) {
+        boolean dbEnabled = containsTTLEnabled && apocConfig.getBoolean(apocTTLEnabledDb);
```

Comment:
Se fai così si semplifica molto:
```suggestion
        boolean dbEnabled = apocConfig.getConfig().getBoolean(apocTTLEnabledDb, enabled);
```